packages/serverless/src/sdk.ts

```diff
@@ -0,0 +1,86 @@
+import { getCurrentHub, initAndBind } from '@sentry/core';
```

Comment:
This whole thing doesn't have to exist once we move it to `@sentry/integrations`

Reply:
made this change in the new pull request.